pkg/graph_addons/topology.go

```diff
@@ -39,6 +39,9 @@ func topologicalSort[K comparable](deps map[K]map[K]graph.Edge[K], less func(K,
 			enqueue(vertex)
 		}
 	}
+	sort.Slice(queue, func(i, j int) bool {
```

Comment:
This has no effect but improves readability by putting the sort next to where all the queuing happens.